src/main/java/ma/boot/springboot/service/mapper/UserMapper.java

```diff
@@ -1,18 +1,17 @@
 package ma.boot.springboot.service.mapper;
 
-import ma.boot.springboot.model.ReviewDto;
 import ma.boot.springboot.model.User;
+import ma.boot.springboot.model.dto.UserRequestDto;
 import org.springframework.stereotype.Component;
 
 @Component
 public class UserMapper {
-    private static final String DEFAULT_PASSWORD = "1111";
 
-    public User mapReviewDtoToUser(ReviewDto dto) {
+    public User mapDtoToUser(UserRequestDto dto) {
         User user = new User();
         user.setProfileName(dto.getProfileName());
         user.setUserId(dto.getUserId());
-        user.setPassword(DEFAULT_PASSWORD);
+        user.setPassword(dto.getPassword());
```

Comment:
maybe this line is not needed - if the default password is set in DatabaseSaver

Reply:
Strange - I have exactly as you put setPassword(dto.getPassword). I wonder why it is not in git?!